web/src/utils/formatting.ts

```diff
@@ -72,13 +73,21 @@ const scalePower = function (maxPower: number | undefined) {
     };
   }
 
-  const thresholds: [number, EnergyUnits][] = [
-    [1e9, EnergyUnits.PETAWATT_HOURS],
-    [1e6, EnergyUnits.TERAWATT_HOURS],
-    [1e3, EnergyUnits.GIGAWATT_HOURS],
-    [1, EnergyUnits.MEGAWATT_HOURS],
-    [1e-3, EnergyUnits.KILOWATT_HOURS],
-  ];
+  const thresholds: [number, EnergyUnits | PowerUnits][] = isPower
+    ? [
+        [1e9, PowerUnits.PETAWATTS],
+        [1e6, PowerUnits.TERAWATTS],
+        [1e3, PowerUnits.GIGAWATTS],
+        [1, PowerUnits.MEGAWATTS],
+        [1e-3, PowerUnits.KILOWATTS],
+      ]
+    : [
+        [1e9, EnergyUnits.PETAWATT_HOURS],
+        [1e6, EnergyUnits.TERAWATT_HOURS],
+        [1e3, EnergyUnits.GIGAWATT_HOURS],
+        [1, EnergyUnits.MEGAWATT_HOURS],
+        [1e-3, EnergyUnits.KILOWATT_HOURS],
+      ];
```

Comment:
In future it would be nice to do something like this, but don't think it's worth it now :)
```suggestion
const unit = isPower ? Units.Power : Units.Energy;
const thresholds: [number, EnergyUnits | PowerUnits][] = [
      [1e9, unit.PETA],
      [1e6, unit.TERA],
      [1e3, unit.GIGA],
      [1, unit.MEGA],
      [1e-3, unit.KILO],
]
```